libs/protobuf/patches/010-rpath.patch

```diff
@@ -1,24 +1,24 @@
 --- a/cmake/install.cmake
 +++ b/cmake/install.cmake
-@@ -16,8 +16,8 @@ foreach(_library ${_protobuf_libraries})
-     $<BUILD_INTERFACE:${protobuf_source_dir}/src>
-     $<INSTALL_INTERFACE:${CMAKE_INSTALL_INCLUDEDIR}>)
+@@ -39,8 +39,8 @@ endif ()
+ 
+ foreach(_library ${_protobuf_libraries})
    if (UNIX AND NOT APPLE)
 -    set_property(TARGET ${_library}
 -      PROPERTY INSTALL_RPATH "$ORIGIN")
-+#   set_property(TARGET ${_library}
++#    set_property(TARGET ${_library}
 +#      PROPERTY INSTALL_RPATH "$ORIGIN")
    elseif (APPLE)
      set_property(TARGET ${_library}
        PROPERTY INSTALL_RPATH "@loader_path")
-@@ -34,8 +34,8 @@ if (protobuf_BUILD_PROTOC_BINARIES)
-     BUNDLE DESTINATION ${CMAKE_INSTALL_BINDIR}
-     COMPONENT protoc)
-   if (UNIX AND NOT APPLE)
--    set_property(TARGET protoc
--      PROPERTY INSTALL_RPATH "$ORIGIN/../${CMAKE_INSTALL_LIBDIR}")
-+#    set_property(TARGET protoc
-+#      PROPERTY INSTALL_RPATH "$ORIGIN/../${CMAKE_INSTALL_LIBDIR}")
-   elseif (APPLE)
-     set_property(TARGET protoc
-       PROPERTY INSTALL_RPATH "@loader_path/../lib")
+@@ -66,8 +66,8 @@ if (protobuf_BUILD_PROTOC_BINARIES)
+   endif ()
+   foreach (binary IN LISTS _protobuf_binaries)
+     if (UNIX AND NOT APPLE)
+-      set_property(TARGET ${binary}
+-        PROPERTY INSTALL_RPATH "$ORIGIN/../${CMAKE_INSTALL_LIBDIR}")
++#      set_property(TARGET ${binary}
++#        PROPERTY INSTALL_RPATH "$ORIGIN/../${CMAKE_INSTALL_LIBDIR}")
+     elseif (APPLE)
+       set_property(TARGET ${binary}
+         PROPERTY INSTALL_RPATH "@loader_path/../lib")
```

Comment:
can this patch be cleaned up?
or upstreamed?

maybe it's fine, but i'm a bit lazy (right now) to try to understand it :)

Reply:
I added that. IIRC it causes linking to OS libraries instead of OpenWrt ones.

Reply:
Actually, as of https://github.com/openwrt/packages/commit/d24229e4415b4a5c4fb7784d20c8748c9a9ebf4b this should not be needed.